cmd/vm_create.go

```diff
@@ -199,14 +199,14 @@ func createVM(deployVM *egoscale.DeployVirtualMachine) (*egoscale.VirtualMachine
 
 func init() {
 	vmCreateCmd.Flags().StringP("zone", "z", "", zoneHelp)
-	vmCreateCmd.Flags().StringP("template", "t", defaultTemplate, "template <name | id>")
+	vmCreateCmd.Flags().StringP("template", "t", defaultTemplate, "template NAME|ID|NAME|ID")
```

Comment:
Erroneous repetition?
```suggestion
	vmCreateCmd.Flags().StringP("template", "t", defaultTemplate, "template NAME|ID")
```